src/ctia/properties.clj

```diff
@@ -1,51 +1,83 @@
-(ns ctia.properties
+(ns ^{:doc "Properties (aka configuration) of the application.
+            Properties are stored in property files.  There is a
+            default file which can be overridden by placing an
+            alternative properties file on the classpath, or by
+            setting system properties."}
+    ctia.properties
   (:refer-clojure :exclude [load])
   (:require [clojure.java.io :as io]
-            [clojure.string :as str])
-  (:import java.util.Properties
-           java.io.BufferedReader
-           java.io.File))
+            [clojure.string :as str]
+            [ctia.lib.map :as map]
+            [schema.coerce :as c]
+            [schema.core :as s]
+            [clj-time.coerce :as coerce])
+  (:import java.util.Properties))
 
-(def property-files ["ctia.properties"
-                     "ctia-default.properties"])
+(def property-files
+  "Property file names, in order of preference"
+  ["ctia.properties"
+   "ctia-default.properties"])
 
-(defonce properties (atom {}))
+(defonce properties
```

Comment:
I wasn't thrilled with properties/properties either, but I feel like "data" is too generic.  I think its okay in this case because almost every ns only cares about this atom (except for ctia.init which has to initialize this).  The other ns'es can just do a :refer [properties] and deref @properties as desired.  I _think_ that is clean.